src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -147,9 +147,9 @@ public void platformMotionDownAABBTest()
                 Cell.WALL, Cell.PLATFORM, Cell.WALL,
                 Cell.WALL, Cell.WALL, Cell.WALL), 3, 4));
         CollisionHandler handler = new CollisionHandler(check);
-        assertTrue(handler.checkLevelAABB(
-                new AABB(new Point(1, 1.1), new Point(2, 2.1)), new Point(0, 1)
-        ));
+        Cell cell = handler.checkLevelAABB(
+                new AABB(new Point(1, 1.1), new Point(2, 2.1)), new Point(0, 1));
+        assertFalse(cell.getType() == Cell.EMPTY);
```

Comment:
Use assertNotEquals instead of == operator combined with assertFalse, == does not work properly for non-objects.
<testing